### DISCUSSION TITLE
8bit or 16bit?

### DISCUSSION THREAD

**Discussion by jho59** [moononournation/Arduino_GFX#514] [General]
Hi,

I'm looking for a library that supports displays with NT35310. 
1. Modules have resistor, you can place at 2 positions to have 8bit parallel or 16bit paralell. But what is Arduino_GFX assuming? Can I use both?

2. In databus-folder I see
- Arduino_ESP32LCD8
- Arduino_ESP32PAR8
- Arduino_ESP32PAR8Q
- Arduino_ESP32PAR8QQ
- Arduino_ESP32PAR8QQQ
The same for 16bit. But no comments or explanations

3. There is no support for SSD1963. Is this, because this controller must be switched to 8bit, 16bit etc by a command?

Bye, Jürgen

**Comment by moononournation** [comment]
No special assumption. If datasheet said it can support 8 bit and 16bit, both you can use.
